src/metalookup/app/api.py

```diff
@@ -57,7 +57,7 @@ async def caching_and_response_time(request: Request, call_next):
 )
 @cache(
     expire=24 * 60 * 60 * 28,
-    key=lambda input, request, response, extra: f"{input.url}-{extra}" if input.splash_response is None else None,
```

Comment:
No - This was done to prevent caching of responses when the received input contained the "splash data to be analyzed". As that field/feature is no longer present we can always cache every response that is not an error :)